src/CRUDTraits/HasModel.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace Backpack\CRUD\CRUDTraits;
+
+
+trait HasModel
+{
+    /**
+     * @param $fqn
+     * @return $this
+     */
+    public function model($fqn)
```

Comment:
`string $fqn`:

1. How does one **pronounce** this? 🙉 
2. Is it better as `$fdqdn`?